static-site/content/blog/2025-01-06-human-metapneumovirus-analysis-and-resources.md

```diff
@@ -0,0 +1,49 @@
+---
+author: "Isabel Joia, Richard Neher"
+date: "2025-01-05"
+title: "Phylogenetic analysis of Human Metapneumovirus"
+sidebarTitle: "Human Metapneumovirus Phylogenetic Analysis and Resources"
+---
+
+We are pleased to announce the availability of the hMPV phylogenetic analysis on [Nextstrain](https://nextstrain.org) and its corresponding [Nextclade dataset](https://docs.nextstrain.org/projects/nextclade/en/stable/user/datasets.html). Below describes an overview of hMPV and its phylogeny. We welcome feedback from hMPV researchers about any aspects of this analysis and dataset.
```

Comment:
I suggest linking to the dataset itself, rather than the docs; also, phrasing suggestion

```suggestion
We are pleased to announce the availability of the hMPV phylogenetic analysis on [Nextstrain](https://nextstrain.org) and its [corresponding Nextclade dataset](nextstrain.org/fetch/data.clades.nextstrain.org/v3/nextstrain/hmpv/all-clades/NC_039199/2024-12-04--17-05-31Z/tree.json). Below, we describe an overview of hMPV and its phylogeny. We welcome feedback from hMPV researchers about any aspects of this analysis and dataset.
```